containers/jenkins-master/config/jobs/github-snappy-integration-tests-cloud/config.xml

```diff
@@ -116,10 +116,80 @@ trap "snappy-cloud-client kill -instance-id $id" EXIT
 go run ./integration-tests/main.go -ip $ip -snappy-from-branch -output-dir $WORKSPACE/$BUILD_TAG -http-proxy http://squid.internal:3128
 
 subunit2junitxml $WORKSPACE/$BUILD_TAG/output/artifacts/results.subunit -o $WORKSPACE/$BUILD_TAG/output/artifacts/results.xml
+
+
+go get github.com/axw/gocov/gocov
+go get github.com/AlekSi/gocov-xml
+BASE_COVERAGE_DIR=$WORKSPACE/$BUILD_TAG/output/artifacts/coverage
+gocov convert $BASE_COVERAGE_DIR/* &gt; $BASE_COVERAGE_DIR/total.json
+cat $BASE_COVERAGE_DIR/total.json | gocov-xml &gt; $BASE_COVERAGE_DIR/total.xml
+
+sed -i &quot;s|&lt;packages&gt;|&lt;sources&gt;&lt;source&gt;&quot;$SRC_PATH&quot;&lt;/source&gt;&lt;/sources&gt;&lt;packages&gt;|&quot; $BASE_COVERAGE_DIR/total.xml
+sed -i &quot;s|filename=\&quot;$SRC_PATH/|filename=\&quot;|g&quot; $BASE_COVERAGE_DIR/total.xml
```

Comment:
mmm because of the `<` and `>` characters I was getting xml errors, I've added a CDATA section and now it seems to be happy